documentation/modules/exploit/winrm/winrm_script_exec.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+Windows Remote Management (WinRM) is a feature of Windows Vista that allows administrators to remotely run management scripts. It handles remote connections by means of the WS-Management Protocol, which is based on SOAP (Simple Object Access Protocol).   This module uses valid credentials to login to the WinRM service and execute a payload. It has two available methods for payload delivery: Powershell 2.0 and VBS CmdStager. This module will check if Poweshell 2.0 is available, and if so then it will use that method. Otherwise it falls back to the VBS CmdStager which is less stealthy.
+IMPORTANT: If targetting an x64 system with the Poweshell method, one must select an x64 payload. An x86 payload will never return. 
```

Comment:
`**bold**`
**bold**

yea just hit enter again to add an extra newline.
In markdown if an item is on a new line (line 3 and 4) when processed its kept on the same line.  If there is an EMPTY line between, its treated as a new line when interpreted.